File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utils/AutoPoseTangent.java
package org.firstinspires.ftc.teamcode.utils;

public class AutoPoseTangent extends AutoPose {
    public double td;

    public AutoPoseTangent(double x, double y, double hd, double td) {
        super(x, y, hd);
        this.td = td;
    }

    public double getTangentRadians() {
        return Math.toRadians(td);
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utils/AutoPoseTangent.kt
package org.firstinspires.ftc.teamcode.utils

class AutoPoseTangent(
    x: Double,
    y: Double,
    hd: Double,
    @JvmField var td: Double,
) : AutoPose(x, y, hd) {
    fun getTangentRadians(): Double {
        return Math.toRadians(td)
    }
}
